asset-manager/README.md

```diff
@@ -200,81 +197,74 @@ To successfully complete this workshop, you need the following:
    
    ![Trigger Assessment](doc-media/trigger-assessment.png)
 
    > **NOTE**: If you are asked to allow the tool access the language models provided by GitHub Copilot Chat, select **Allow** to proceed.
 
 1. After each step, please manually input "continue" to confirm and proceed.
 1. Wait for the assessment to be completed and the report to be generated.
-1. Review the **Summary** report. Select **Propose Solution** to view the proposed solutions for the issues identified in the summary report.
-1. For this workshop, deselect all solutions and select **Migrate to Azure Database for PostgreSQL (SDK on Public Cloud)** in the Solution report, then select **Confirm Solution**.
+1. Review the **Assessment** report. Select the **Issues** tab to view the proposed solutions for the issues identified in the report.
+1. For this workshop, select **Migrate to Azure Database for PostgreSQL (Spring)** in the Solution list, then click **Run Task**.
    ![Confirm Solution](doc-media/confirm-postgresql-solution.png)
-1. In the Migrate report, click **Migrate**.
 
-### Migrate to Azure Database for PostgreSQL Flexible Server using Predefined Formula
+### Migrate to Azure Database for PostgreSQL Flexible Server using Predefined Tasks
 
-1. After clicking the Migrate button in the Solution Report, Copilot chat window will be opened with Agent Mode.
-1. In Copilot chat window, click **Continue** repeatedly to confirm each tool action. The Copilot Agent uses various tools to facilitate application modernization. Each tool's usage requires confirmation by clicking the `Continue` button.
-1. After each step, please manually input "Continue" to confirm and proceed.
-1. Click **Continue** to confirm to run **Java Application Build-Fix** tool. This tool will attempt to resolve any build errors, in up to 10 iterations.
-1. After the Build-Fix tool begins, click **Continue** to proceed and show progress and migration summary.
+1. After clicking the **Run Task** button in the Solution Report, Copilot Chat panel will be opened with Agent Mode.
+1. The Copilot Agent will firstly analyse the project and generate a migratin plan. During the progress, please click **Continue** repeatedly to confirm each tool action. The Copilot Agent uses various tools to facilitate application modernization. Each tool's usage requires confirmation by clicking the `Continue` button.
+1. After the plan is generated, Copilot chat will stop with two generated files: **plan.md** and **progress.md**. Please manually input "Continue" or "Proceed" in the chat to confirm the plan and proceed its following actions to execute the plan.
+1. Click **Allow** repeatedly to allow the git operations for version control during the migration process.
```

Comment:
In the latest `main` to be released this week, we don't need to confirm for Git operations now.